src/zkevm_specs/evm/execution/extcodecopy.py

```diff
@@ -0,0 +1,56 @@
+from ..instruction import Instruction, Transition
+from ...util.param import EXTRA_GAS_COST_ACCOUNT_COLD_ACCESS
+from ...util import N_BYTES_MEMORY_ADDRESS, FQ
+from ..table import AccountFieldTag, CallContextFieldTag, CopyDataTypeTag
+
+
+def extcodecopy(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    address, memory_offset_word, code_offset, size_word = (
```

Comment:
Just a nitpick, but we usually use a `_rlc` suffic for RLC values. So in this case you would have `memory_offset_rlc` and `size_rlc`.

Feel free to use or ignore this suggestion.

Reply:
I would love to follow the code convention :) I fixed in [44faa4b](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/257/commits/44faa4b2ff67ca3d56fc362b3162f990a0f8ab59) :)